babel-watch.js

```diff
@@ -247,18 +246,21 @@ function restartApp() {
     }
   }
 
-  // Support for --debug option
   const runnerExecArgv = process.execArgv.slice();
-  if (program.debug) {
-    runnerExecArgv.push('--debug=' + program.debug);
-  }
-  // Support for --inspect option
-  if (program.inspect) {
-    runnerExecArgv.push('--inspect');
-  }
-  // Support for --debug-brk
-  if(program.debugBrk) {
-    runnerExecArgv.push('--debug-brk');
+  const unknownOptions = program.parseOptions(program.normalize(process.argv.slice(2))).unknown;
+
+  if (unknownOptions.length > 0) {
+    for (let i = 0, len = unknownOptions.length; i < len; ++i) {
+      // If this unknown option is a '--' option and the next option looks like it might
+      // be an argument for this option, we format it as --option=argument and pass it on.
+      if (unknownOptions[i].indexOf('--') === 0 &&
+          unknownOptions[i + 1] &&
+          unknownOptions[i + 1].indexOf('-') !== 0) {
+        runnerExecArgv.push(unknownOptions[i] + '=' + unknownOptions[++i]);
```

Comment:
It can't be so easy to solve this problem just by using space instead of `=`, I think.

Following POSIX docs https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html

> To specify an argument for a long option, write ‘--name=value’. This syntax enables a long option to accept an argument that is itself optional.

node options:

```
❯ node --help
Usage: node [options] [ -e script | script.js ] [arguments]
       node debug script.js [arguments]

Options:
  -v, --version         print Node.js version
  -e, --eval script     evaluate script
  -p, --print           evaluate script and print result
  -c, --check           syntax check script without executing
  -i, --interactive     always enter the REPL even if stdin
                        does not appear to be a terminal
  -r, --require         module to preload (option can be repeated)
  --no-deprecation      silence deprecation warnings
  --trace-deprecation   show stack traces on deprecations
  --throw-deprecation   throw an exception anytime a deprecated function is used
  --no-warnings         silence all process warnings
  --trace-warnings      show stack traces on process warnings
  --trace-sync-io       show stack trace when use of sync IO
                        is detected after the first tick
  --track-heap-objects  track heap object allocations for heap snapshots
  --prof-process        process v8 profiler output generated
                        using --prof
  --zero-fill-buffers   automatically zero-fill all newly allocated
                        Buffer and SlowBuffer instances
  --v8-options          print v8 command line options
  --v8-pool-size=num    set v8's thread pool size
  --tls-cipher-list=val use an alternative default TLS cipher list
  --openssl-config=path load OpenSSL configuration file from the
                        specified path
  --icu-data-dir=dir    set ICU data load path to dir
                        (overrides NODE_ICU_DATA)
  --preserve-symlinks   preserve symbolic links when resolving
                        and caching modules.

Environment variables:
NODE_PATH                ':'-separated list of directories
                         prefixed to the module search path.
NODE_DISABLE_COLORS      set to 1 to disable colors in the REPL
NODE_ICU_DATA            data path for ICU (Intl object) data
NODE_REPL_HISTORY        path to the persistent REPL history file

Documentation can be found at https://nodejs.org/
```

I've no idea how to solve it. Maybe better to use other parser instead of Commander.